src/style.css

```diff
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  width: 91px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-right: 9px;
+}
+
+.stars_star {
+  width: 15px;
+  height: 15px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 div:nth-child(n) {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 div:nth-child(n + 2) {
+  background-image: url(images/star.svg);
+}
+
+.stars--2 div:nth-child(n + 3) {
+  background-image: url(images/star.svg);
+}
+
+.stars--3 div:nth-child(n + 4) {
+  background-image: url(images/star.svg);
+}
+
+.stars--4 div:nth-child(n + 5) {
+  background-image: url(images/star.svg);
+}
```

Comment:
Don't style using tags. you can write simply `.stars--4 :nth-child(n + 5)`

Also, you can combine these selectors like this:
```
selector1,
selector2,
selector3 {
 ...styles
}
```